models/staging/zendesk_staging/stg_rag_zendesk__ticket.sql

```diff
@@ -54,7 +54,7 @@ final as (
         recipient,
         requester_id,
         status,
-        subject,
+        subject as title,
```

Comment:
This needs to be documented as a breaking change in the CHANGELOG

Reply:
added